pkgs/development/python-modules/cyipopt/default.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  cython,
+  ipopt,
+  numpy,
+  pkg-config,
+  pytestCheckHook,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "cyipopt";
+  version = "1.5.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "mechmotum";
+    repo = "cyipopt";
+    rev = "v${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```
To prevent branch/tag mismatch.